libs/agno/agno/run/v2/workflow.py

```diff
@@ -486,6 +492,9 @@ def to_dict(self) -> Dict[str, Any]:
                     flattened_responses.append(step_response.to_dict())
             _dict["step_responses"] = flattened_responses
```

Comment:
What is step responses then? 

How above we have
- `step_results` instead of `step_responses`
- `step_member_runs` instead of `member_runs`

wdyt?

Reply:
Step responses are just this now, stores content of that step result etc. 
<img width="784" height="983" alt="image" src="https://github.com/user-attachments/assets/884ee1f1-335a-4442-b769-613af483cbd3" />

> How above we have

> step_results instead of step_responses
> step_member_runs instead of member_runs

yea i like it